pkg/cmd/util/clientcmd/errors.go

```diff
@@ -103,6 +123,12 @@ func detectReason(err error) int {
 		case strings.Contains(err.Error(), "tls: oversized record received"):
 			return tlsOversizedRecordReason
 		}
+		switch err.(type) {
```

Comment:
go ahead and add a case for UnknownAuthorityError here and return certificateAuthorityUnknownReason, just in case they change the text in the future